crypto/s390x_arch.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2017 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef S390X_ARCH_H
+# define S390X_ARCH_H
+
+/*
+ * The elements of OPENSSL_s390xcap_P are the doublewords returned by the STFLE
```

Comment:
Trouble with using "doubleword" is that its meaning varies from platform to platform, and meaning on most popular platform is actually 32-bit word. For this reason I'd like to suggest using less ambiguous term. Shall we say "64-bit words"? And in order to minimize repetition one can write "If STFLE returns fewer *data* ..."

Reply:
Im probably too caught up in POP-notation :)  will fix that.